stories/pdf-viewer/pdf-viewer.stories.js

```diff
@@ -0,0 +1,196 @@
+import '../../dist/dialog.css';
+import '../../dist/pdf-viewer.css';
+import '../../dist/message-page.css';
+import '../../dist/title.css';
+
+export default {
+    title: 'Components/PDF Viewer',
+    parameters: {
+        description: `
+The PDF viewer component can be either displayed within a page or inside of a dialog.
+
+##Usage
+**Use the PDF viewer if:**
+
+- You want your app to display PDF files on all devices and platforms.
+- You want the users of your app to be able to preview their documents as PDF files right beside your app.
+- You need to ensure the consistent behavior of PDF files across your app.
+
+**Do not use the PDF viewer if:**
+
+- You need to provide an interactive PDF file (such as a data input form).
+
+`,
+        tags: ['f3', 'a11y']
+    }
+};
+
+export const dialog = () => `
+<div class="fd-dialog-docs-static fd-pdf-viewer fd-dialog fd-dialog--active">
+    <div class="fd-pdf-viewer__dialog fd-dialog__content fd-dialog__content--l" style="min-height: 28rem;" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
+        <header class="fd-dialog__header fd-bar fd-bar--header">
+            <div class="fd-bar__left">
+                <div class="fd-bar__element">
+                    <h2 class="fd-title fd-title--h5" id="dialog-title-1">
+                        PDF Viewer title
+                    </h2>
+                </div>
+            </div>
+        </header>
+        <div class="fd-dialog__body">
```

Comment:
Since this can become scrollable, should we add `tabindex="0"` here?
```suggestion
        <div class="fd-dialog__body" tabindex="0">
```

![image](https://user-images.githubusercontent.com/43764832/93401592-5e777c00-f837-11ea-9b7e-70264a436860.png)

Reply:
I removed the scrollbar instead and made a note that the heights should not allow the dialog to scroll. We'd like to avoid putting tabindex on non-interactive items as much as necessary and it'd be really annoying to have to tab out, scroll the dialog down, then tab back into the iframe to access the rest.